components/Cart/interfaces.rst

```diff
@@ -0,0 +1,102 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_cart_model_cart-interface:
+
+CartInterface
+~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Cart.
+
+.. note::
+    This interface extends the :ref:`component_order_model_order-interface` |br|
+    For more detailed information go to `Sylius API's CartInterface`_.
+
+.. _Sylius API's CartInterface: http://api.sylius.org/Sylius/Component/Cart/Model/CartInterface.html
+
+.. _component_cart_model_cart-item-interface:
+
+CartItemInterface
+~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single CartItem.
+
+.. node::
+    This interface extends the :ref:`component_order_model_order-item-interface` |br|
+    For more detailed information go to `Sylius API's CartItemInterface`_.
+
+.. _Sylius API's CartItemInterface: http://api.sylius.org/Sylius/Component/Cart/Model/CartItemInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_cart_provider_cart-provider-interface:
+
+Cart provider
+~~~~~~~~~~~~~
+
+A cart provider retrieves existing cart or create new one based on storage. To characterize an object which is a **Provider**,
+it needs to implement the ``CartProviderInterface``.
```

Comment:
I'm not really satisified with such descriptions of interfaces. Thay should be a bit more... human :) 
 Maybe : 
`This interface gives provides a possibility to retrieve an existing cart or create a new one basing on the storage.` ?